server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
Hi @tombuildsstuff I don't want to block the development, but I think if this is gonna break us, it is gonna probably break other users too.
I made a PR for the alternative approach I would prefer for making dependency requirements cleaner, which shouldn't break people that use hobknob as a dependency.